test-framework/go.mod

```diff
@@ -0,0 +1,96 @@
+module github.com/onflow/cadence/test
+
+go 1.18
+
+require (
+	github.com/onflow/cadence v0.24.6
+	github.com/onflow/flow-emulator v0.33.4-0.20220719174752-88332a9af217
```

Comment:
This makes me wary of putting another distinct module in the Cadence repo along with Cadence itself and the Language Server. Have we given any thought to putting this in a different repo? We could move this to the same repo as the linter, for example, and maybe rename it from `cadence-lint` to `cadence-tooling`?

Reply:
Yea, this can go to a separate repo eventually. Kept it inside cadence for now, because it makes it easier during the initial phase (i.e: needs a lot of coordinated changes between the test-stdlib and this module).

Agree that all tooling should go to a separate repo/repos 👍